common/scripts/ensure-consistent-versions.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why is this script here?
I thought it was part of your PR @prprabhu-ms  ?
cc @mgamis-msft

Reply:
Not sure how this happened after I merged main to my branch but I have now pruned these other changes from this PR.